### DISCUSSION TITLE
Action fails after 20 minutes without log

### DISCUSSION THREAD

**Discussion by leezu** [community/community#26977] [Actions]
<p><a href="https://github.com/leezu/gluon-nlp/runs/676113630?check_suite_focus=true" rel="nofollow noopener">https://github.com/leezu/gluon-nlp/runs/676113630?check_suite_focus=true</a></p>
<p>Log only contains</p>
<pre><code>2020-05-14T21:41:25.9791592Z ##[section]Starting: Request a runner to run this job
2020-05-14T21:41:26.6189821Z Can't find any online and idle self-hosted runner in current repository that matches the required labels: 'ubuntu-latest'
2020-05-14T21:41:26.6189857Z Can't find any online and idle self-hosted runner in current repository's account/organization that matches the required labels: 'ubuntu-latest'
2020-05-14T21:41:26.6189876Z Found online and idle hosted runner in current repository's account/organization that matches the required labels: 'ubuntu-latest'
2020-05-14T21:41:26.7539120Z ##[section]Finishing: Request a runner to run this job
</code></pre>

**Comment by weide-zhou** [comment, marked as answer]
<p>Hi <a class="mention" href="/leezu">@leezu</a> ,</p>
<p>There is no log output due to <strong>no disk space left</strong>  on your ubuntu hosted runner, in your workflow <a href="https://github.com/leezu/gluon-nlp/actions/runs/104219185" rel="nofollow noopener">annotation</a> displays as below.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/1/15c85d121b05ffefc3bdaf78825da01b5052a5c2.png" data-download-href="/uploads/short-url/36HiPTccm4ThpVyJVH0tGGZTmvg.png?dl=1" title="Nospace.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/52772761/181101031-92d34a16-8b28-4083-9ce2-1d6b9aec847d.png" alt="Nospace.png" data-base62-sha1="36HiPTccm4ThpVyJVH0tGGZTmvg" width="690" height="396" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/1/15c85d121b05ffefc3bdaf78825da01b5052a5c2_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Nospace.png</span><span class="informations">778×447 21.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>This is a known issue which is being tracking in issue ticket here: <a href="https://github.com/actions/virtual-environments/issues/709" rel="nofollow noopener">https://github.com/actions/virtual-environments/issues/709</a>, you can monitor and check if there’s any update.</p>
<p>In the meanwhile, you can try the <a href="https://github.com/actions/virtual-environments/issues/709#issuecomment-612569242" rel="nofollow noopener">workaround</a> mentioned in the ticket or use a self-hosted runner instead.</p>
<p>Thanks.</p>
